RabbitMQ.pm

```diff
@@ -81,26 +83,30 @@ will exit with error code 141
 
 =head2 disconnect()
 
-Causes the connection to RabbitMQ to be torn down.
+Disconnect from the RabbitMQ server.
 
 =head2 get_server_properties()
 
-Get a hashref of server properties (these may vary, you should C<Data::Dumper> to inspect). They will be provided by the RabbitMQ server to which you are connected.
+Get a reference to hash of server properties (these may vary, you should use C<Data::Dumper> to inspect). Properties will be provided for the RabbitMQ server to which you are connected.
```

Comment:
I added (hashref) back to the docs.  I'm a little confused how hashref adds clarity over "reference to hash" so I split the difference left them both in.